src/RA_Dlg_AchEditor.cpp

```diff
@@ -1777,8 +1807,7 @@ void Dlg_AchievementEditor::GetListViewTooltip()
     m_sTooltip = NativeStr(oss.str());
 }
 
-[[gsl::suppress(con.4)]] void Dlg_AchievementEditor::UpdateSelectedBadgeImage(const std::string& sBackupBadgeToUse)
-{
+[[gsl::suppress(con .4)]] void Dlg_AchievementEditor::UpdateSelectedBadgeImage(const std::string& sBackupBadgeToUse) {
```

Comment:
this should be changed to use the GSL_SUPPRESS_CON4 macro

Reply:
Sure